### PR TITLE
Fix some doc strings in the volume package

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

**Review by tonistiigi — APPROVED** [review]

File: volume/service/service.go [hunk 1]
```diff
@@ -54,6 +55,11 @@ func (s *VolumesService) GetDriverList() []string {
 }
 
 // Create creates a volume
+// If the caller s creating this volume to be consumed immediately, it is expected that the caller specifies a reference ID.
```
Comment (thaJeztah) [inline volume/service/service.go]: `s/caller s/caller is/`? (ducks)

File: volume/service/service.go [hunk 2]
```diff
@@ -68,7 +74,7 @@ func (s *VolumesService) Create(ctx context.Context, name, driverName string, op
 	return &apiV, nil
 }
 
-// Get gets a volume
+// Get gets details about a volume
```
Comment (thaJeztah) [inline volume/service/service.go]: nit: gets -> returns perhaps

File: volume/service/service.go [hunk 3]
```diff
@@ -118,6 +136,7 @@ func (s *VolumesService) Release(ctx context.Context, name string, ref string) e
 }
 
 // Remove removes a volume
+// An error will be returned if the volume is still referenced.
```
Comment (thaJeztah) [inline volume/service/service.go]: (suggestion) will be -> is

File: volume/service/store.go [hunk 4]
```diff
@@ -182,7 +182,7 @@ func (s *VolumeStore) purge(ctx context.Context, name string) error {
 	return nil
 }
 
-// VolumeStore is a struct that stores the list of volumes available and keeps track of their usage counts
+// VolumeStore is responsible for storing and referencing counting volumes.
```
Comment (thaJeztah) [inline volume/service/store.go]: referencing counting -> reference counting

**Review by thaJeztah — APPROVED** [review]
found some nits/suggestions, but I'm ok merging as-is; let me know if you want to update

LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM, thx!

**Review by AkihiroSuda — APPROVED** [review]

**Comment by thaJeztah** [comment]
failure is unrelated; merging

**Pull request merged**